src/components/Avatar/Avatar.component.tsx

```diff
@@ -39,6 +40,24 @@ export const CAMERA = {
     }
   }
 };
+export interface SpawnState {
+  onMountEffect?: {
```

Comment:
`onLoadedEffect` & `onLoadedAnimation` reads more intuitive to me.

Reply:
no strong opinions, changing!